source/pdps-group-replication.rst

```diff
@@ -0,0 +1,233 @@
+
+.. image:: _static/images/group-replication-1.png
+
+
+
+.. _failovers:
+
+Failovers
+--------------------
+
+A database failure or configuration change that requires a restart should not affect the stability of the database infrastructure, if it is properly planned and architected. Failovers are an integral part of a stability strategy and aligning the business requirements for availability and uptime with failover methodologies is critical. 
+
+The following are the 3 main types of failovers that can occur in database environments:
+
+* **Planned failover**. A planned failover is a failover that has been scheduled in advance or occurs at a regular interval. There can be many reasons for planned failovers including patching, large data operations, retiring existing infrastructure, or simply to test the failover strategy.
+
+* **Unplanned failover**. An unplanned failover is what occurs when a database has unexpectedly become unresponsive or experiences instability. This could also include emergency changes that do not fall under the planned failover cadence or scheduling parameters. Unplanned failovers are generally considered higher risk operations due to the high stress and high potential for either data corruption or data fragmentation.
+
+* **Regional or disaster recovery (DR) failover**. Unplanned failovers still work with the assumption that additional database infrastructure is immediately available and in a usable state. In a regional or DR failover, it is assumed that there is a large scale infrastructure outage which requires the business to move its operations away from its current availability zone.
+
+Maintenance windows
+--------------------------
+
+Major vs Minor maintenance
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Although it may not be obvious at first, not all maintenance activities are created equal and do not have the same dependencies. It is good to separate maintenance that demands downtime or failover from maintenance that can be done without impacting those important stability metrics. When defining these maintenance dependencies, there can be a change in the actual maintenance process that allows for a different cadence.
+
+Maintenance without service interruption
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is possible to cover both major and minor maintenance without service interruption with rolling restart and using proper version upgrade.
+
+Uptime
+--------------------------
+
+When referring to database stability, uptime is likely the largest indicator of stability and oftentimes is the most obvious symptom of an unstable database environment. Uptime is composed of 3 key components and, contrary to common perception, is based on what happens when the database software is not able to take incoming requests rather than maintaining the ability to take requests with errors.
```

Comment:
and often 
oftentimes is not a common phrase

Reply:
fixed